apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -82,32 +84,18 @@ type FilesComponentProps = {
   moveFilePrompt: moveFilePromptType;
   moveFolderPrompt: moveFolderPromptType;
   newFilePrompt: newFilePromptType;
-  newFolderPrompt: newFolderPromptType;
   renameFilePrompt: renameFilePromptType;
   renameFolderPrompt: renameFolderPromptType;
   setFileType: setFileType;
   appName?: string;
 };
 
-// given a promise returned from DialogManager.showDialog({type : DialogType.GenericPrompt}), will return the input
-// that was typed in by the user.
-// Note that if the user did not press the `confirm` button, then an empty string will be returned instead.
-const extractInput = (promiseResults: DialogClosePromiseReturnType): string => {
-  const {type, args} = promiseResults;
-  if (type === 'confirm') {
-    return args as string;
-  }
-
-  return '';
-};
-
```

Comment:
this function was moved to lab2/views/dialogs/extractDialogClosePromiseInput.ts`